docs/nnx/surgery.md

```diff
@@ -42,19 +42,19 @@ class TwoLayerMLP(nnx.Module):
     return self.linear2(x)
 ```
 
-## Pythonic module manipulations
+## Pythonic module manipulation
 
-Model surgery is easiest when you already have a fully fleshed-out model loaded with correct parameters, and you don't intend to change your model definition code.
+Doing model surgery is easiest when you already have a fully fleshed-out model loaded with correct parameters, and you don't intend to change your model definition code.
 
-You can make a variety of pythonic operations on its submodules, like swapping in/out, sharing modules/weights, monkeypatching, etc. See a few code examples below.
+You can perform a variety of Pythonic operations on its sub-modules, such as swapping in/out, sharing modules/weights, monkey-patching, and so on. Below are examples of sub-module sharing, module sharing, variable sharing, and monkey-patching:
```

Comment:
> sub-module sharing, module sharing, variable sharing, and monkey-patching
Repeating this twice seems verbose. Can we do this instead:

> You can perform a variety of Pythonic operations on its sub-modules, such as sub-module swapping, module sharing, variable sharing, and monkey-patching:

Reply:
SGTM!